cpplite/Changelog.txt

```diff
@@ -0,0 +1,29 @@
+Changes in v0.3:
+- Parallel blob uploading & downloading
+- Proxy support
+- Configurable retry policy
+- Default retry count is changed from 25 to 3
```

Comment:
So, do we need all these files in our repo?

Reply:
As we have taken cpplite code base as subtree, whatever they have in their repo will show up in ours too. We have not added any of such file explicitly.